src/utilities.py

```diff
@@ -411,18 +411,20 @@ def break_long_message(phrases, joinstr = " "):
 
 #completes a partial nickname or string from a list
 def complete_match(string, matches):
-    num_matches = 0
-    bestmatch = string
+    possible_matches = set()
     for possible in matches:
         if string == possible:
-            return string, 1
+            return {string}
         if possible.startswith(string) or possible.lstrip("[{\\^_`|}]").startswith(string):
-            bestmatch = possible
-            num_matches += 1
-    if num_matches != 1:
-        return None, num_matches
-    else:
-        return bestmatch, 1
+            possible_matches.add(possible)
+    return sorted(list(possible_matches))
```

Comment:
I switched the statement above it to a list.

Using lists here so the matches would be in order.